server/jetstream_cluster_2_test.go

```diff
@@ -6361,24 +6361,30 @@ func TestJetStreamClusterEncryptedDoubleSnapshotBug(t *testing.T) {
 	require_NoError(t, err)
 }
 
-func TestJetStreamClusterRePublishUpdateNotSupported(t *testing.T) {
+func TestJetStreamClusterRePublishUpdateSupported(t *testing.T) {
 	test := func(t *testing.T, s *Server, stream string, replicas int) {
-		nc := natsConnect(t, s.ClientURL())
+		nc, js := jsClientConnect(t, s)
 		defer nc.Close()
 
-		cfg := &StreamConfig{
-			Name:     stream,
-			Storage:  MemoryStorage,
+		cfg := &nats.StreamConfig{
+			Name:     stream + "_1",
+			Storage:  nats.MemoryStorage,
 			Replicas: replicas,
+			Subjects: []string{"foo.>"},
 		}
-		addStream(t, nc, cfg)
+		_, err := js.AddStream(cfg)
+		require_NoError(t, err)
 
-		cfg.RePublish = &RePublish{
-			Source:      ">",
-			Destination: "bar.>",
+		cfg2 := &nats.StreamConfig{
```

Comment:
This even used?

Reply:
The `cfg2` stream is added below, but not modified afterwards. Only `cfg` is changed.

Reply:
Why is that stream there is what I was asking..

Reply:
Oh, I had to put it in so that I could subscribe to `bar.>` to check whether the messages were being republished there or not. Without it, the subscribe errors with `nats: no stream matches subject`.

Reply:
Just do low level NATS sub to check if they are being republished. No need for 2nd stream etc.

Reply:
That seems like a better idea, done.